lemminx-liberty/src/test/java/io/openliberty/LibertyHoverTest.java

```diff
@@ -74,8 +74,23 @@ public void testXSDSchemaHover() throws BadLocationException, IOException {
 
                 XMLAssert.assertHover(serverXML, serverXMLURI, "Defines how the server loads features." + //
                                 System.lineSeparator() + System.lineSeparator() + //
-                                "Source: [server-cached-24.0.0.6.xsd](" + serverXSDURI + ")", //
+                                "Source: [server-cached-24.0.0.8.xsd](" + serverXSDURI + ")", //
                                 r(1, 8, 1, 22));
         }
 
+        @Test
+        public void testPlatformHover() throws BadLocationException {
+
+                String serverXML = String.join(newLine, //
+                        "<server description=\"Sample Liberty server\">", //
+                        "       <featureManager>", //
+                        "               <platform>j|avaee-6.0</platform>", //
+                        "       </featureManager>", //
+                        "</server>" //
+                );
+                XMLAssert.assertHover(serverXML, serverXMLURI,
+                        "Description: This platform resolves the Liberty features that support the Java EE 6.0 platform.",
```

Comment:
Since we are hard-coding the check for platform for providing the description, can you go ahead and add two more test methods here to cover jakartaee and microprofile?

Reply:
Done